From: Vincent Tremblay <vincent@vtremblay.dev>
To: unlisted-recipients:; (no To-header on input)
Cc: vincent@vtremblay.dev, "Rob Herring" <robh+dt@kernel.org>,
	"Krzysztof Kozlowski" <krzysztof.kozlowski+dt@linaro.org>,
	"Mark Brown" <broonie@kernel.org>,
	"Guenter Roeck" <linux@roeck-us.net>,
	"Stanislav Jakubek" <stano.jakubek@gmail.com>,
	"Marcello Sylvester Bauer" <sylv@sylv.io>,
	"Marek Vasut" <marex@denx.de>,
	"Samuel Holland" <samuel@sholland.org>,
	"Greg.Schwendimann@infineon.com" <Greg.Schwendimann@infineon.com>,
	"Conor Dooley" <conor.dooley@microchip.com>,
	"Pali Rohár" <pali@kernel.org>,
	devicetree@vger.kernel.org, linux-kernel@vger.kernel.org,
	linux-spi@vger.kernel.org
Subject: [PATCH v2 2/2] spidev: Add Silicon Labs EM3581 device compatible
Date: Mon, 26 Dec 2022 21:35:48 -0500	[thread overview]
Message-ID: <20221227023550.569547-2-vincent@vtremblay.dev> (raw)
In-Reply-To: <20221227023550.569547-1-vincent@vtremblay.dev>

Add compatible string for Silicon Labs EM3581 device.

Signed-off-by: Vincent Tremblay <vincent@vtremblay.dev>
---
Changes in v2:
 - Add missing documentation in trivial-devices

 drivers/spi/spidev.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/spi/spidev.c
+++ b/drivers/spi/spidev.c
@@ -691,6 +691,7 @@ static const struct spi_device_id spidev
 	{ .name = "m53cpld" },
 	{ .name = "spi-petra" },
 	{ .name = "spi-authenta" },
+	{ .name = "em3581" },
 	{},
 };
 MODULE_DEVICE_TABLE(spi, spidev_spi_ids);
@@ -705,6 +706,7 @@ static const struct of_device_id spidev_
 	{ .compatible = "menlo,m53cpld" },
 	{ .compatible = "cisco,spi-petra" },
 	{ .compatible = "micron,spi-authenta" },
+	{ .compatible = "silabs,em3581" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, spidev_dt_ids);
